python/tvm/topi/arm_cpu/mprofile/dsp/conv1d.py

```diff
@@ -105,7 +106,12 @@ def conv1d_nwc_dsp_compute(cfg, data, kernel, strides, padding, dilation, out_dt
         policy="factors",
         num_outputs=2,
         # TODO: check case with in_channels.value % 4 != 0 with AutoTVM
-        filter=None if cfg.is_fallback else lambda x: x.size[-1] % 4 == 0,
```

Comment:
I have a better suggestion for this which I was gonna try on dense layer, let me try it today and get back to you